server/src/immich/api-v1/asset/asset-repository.ts

```diff
@@ -136,6 +137,7 @@ export class AssetRepository implements IAssetRepository {
       order: {
         fileCreatedAt: 'DESC',
       },
+      withDeleted: dto.withDeleted,
```

Comment:
I believe this is only ever used by the mobile app. We can probably just default this to true, instead of making it configurable?